package.json

```diff
@@ -8,8 +8,8 @@
     "build": "tsc && vite build",
     "preview": "vite preview",
     "prepare": "./scripts/prepare",
-    "lint": "eslint ./src/ --fix",
-    "lint:ci": "eslint ./src/",
+    "lint:ci": "eslint . --ext .ts,.tsx",
+    "lint": "npm run lint:ci -- --fix",
     "spell": "cspell ./src/**/*",
```

Comment:
we can but it's not really that important or useful
because we should't care about for example that eslint has a rule with a word in it that is not recognized by cspell

Reply:
I agree with @erfan-goodarzi